CONTRIBUTING.md

```diff
@@ -233,6 +237,58 @@ Once your pull request has been reviewed, if all conditions above have been met
+
+- Review and merge code and documentation.
+- Help triaging bugs and testing pull requests.
+- Make well formed pull requests.
+- Have a sense of duty about the project.
+- Play well with others, are respectful, show gratitude.
+
+If you want to become a maintainer, we expect you to:
+
+- Review and test pull requests submitted by others.
+- Encourage and ensure design remains an integral part of the review process and pull in designers for review as needed (you can leverage @patternfly/pf-<framework>-design if there is no known associated designer).
+- Maintain sustained activity versus sporadic.
+- Support users and other developers on [PatternFly Slack](https://patternfly.slack.com/) and the [mailing list](mailto:patternfly@redhat.com).
+
+Other things that are nice to do:
+
+- Participate in (or even initiate) real-world events such as user/developer meetups, papers/talks at conferences, etc.
+
+### How do I become a maintainer?
+
+One person has to nominate you to the group of existing maintainers. The person who nominates you has to:
+
+- Submit ten examples that prove this person behaves like a maintainer.
+- Explain how the nominee is involved in the community and cares about the future of the project
+
+This nomination is public and should be sent to the PatternFly mailing list. After the nomination is submitted, two other maintainers have to second the nomination. If no one objects in one week, the nomination is accepted.
+
+Such objections may happen in public on the nomination e-mail thread. However, not everyone is comfortable giving objections publicly. Therefore, it is acceptable for other maintainers to raise their concerns with the sponsor and/or other maintainers privately if they wish to do so. The sponsor is expected to update the nomination e-mail thread to show that it is on hold pending private concerns.
+
+Regardless, while objections are discussed either in private or public, the nomination is on hold until the objections are resolved or the nomination is rejected. In the event of a failed nomination, the sponsor (as part of the discussing group) will know the grounds for the rejection, and can pass along constructive feedback to the candidate. Care should be taken to do this sensitively.
+
+### How do I lose maintainers status?
+
+If you are inactive in the community for one year, we will remove you from the maintainers list and revoke your permission, but we will make a mention of you on a  list of previous maintainers.
+
+In the event that a maintainer continues to disregard good citizenship (or actively disrupts the project), we may need to revoke that person’s status. The process is the same as for nominating a new maintainer: someone suggests the revocation with a good reason, two people second the motion, and a vote may be called if consensus cannot be reached. We hope that’s simple enough, and that we never have to test it in practice.
+
```

Comment:
The process for nominating a person sounds good, but the same process for revoking? I can see that revoking a persons privileges publicly could get ugly. This should probably be done more discreetly so people are not publicly shamed.

Reply:
Similar to Dan's comment, if I can privately object to a nomination, then should I be able to privately request that their status be removed? If that's the case, who would I send my request to?